prody/proteins/tabulated_energies.txt

```diff
@@ -398,3 +398,120 @@ PRO LYS 0.3	-0.5	0.0
 PRO ARG -0.17	-2.43	0.0
 PRO HIS 0.33	-1.8	0.0
 PRO PRO -0.03	-0.83	0.0
+GLY HSE 0.01	-2.47	0.0
```

Comment:
maybe you can have a check to replace HSE or HSD or HSP with HIS before querying this, rather than making it look like we have data specifically for these protonation versions

We should probably also check with Carlos what histidine protonation he did use

Reply:
You are right, but we will not have HSP, HSE, or HSP data anyway. From Carlos, we will have only one type, and from Ivet paper, too. Maybe I can add a comment that we treat histidines HSP, HSE and HSD as HIS, i.e., without distinguishing differences in protonation.

Reply:
> maybe you can have a check to replace HSE or HSD or HSP with HIS before querying this, rather than making it look like we have data specifically for these protonation versions

I can do it.

Reply:
> You are right, but we will not have HSP, HSE, or HSP data anyway. From Carlos, we will have only one type, and from Ivet paper, too. Maybe I can add a comment that we treat histidines HSP, HSE and HSD as HIS, i.e., without distinguishing differences in protonation.

Yes, a comment in docs and code would be good